### ISSUE TITLE
Elapsed time gets reset in the midle of a build

### ISSUE THREAD

**Issue by AndersHoglund** [travis-ci/travis-ci#6837]
We have a huge build consisting of 56 jobs currently. Today all of a sudden the elapsed time counter started acting up in a weird way, about 10 minutes into the build it started over from 0. I have been playing with our .travis.yml all day trying to improve performance by minimizing caching etc. Is it possible that I might gave done something  in there to affect timers ? When all jobs are finished it says:
![travis_time](https://cloud.githubusercontent.com/assets/15121917/20076148/997ec7c4-a537-11e6-8429-62e068086c20.jpg)
The true real elapsed time is about 23minutes. 

This is the build: https://travis-ci.org/betaflight/betaflight/builds/173989644
And this is the source: https://github.com/betaflight/betaflight/pull/1486

**Comment by BanzaiMan** [comment]
"Total time" refers to the sum of all the times that the jobs took. While the jobs are running, it will show the clock time since the first job started running. When the jobs finish, it will switch to the sum.

This mismatch is discussed in https://github.com/travis-ci/travis-ci/issues/1486.

I'm closing this.

**Comment by AndersHoglund** [comment]
Hey, total time is probably correct. It is the ELAPSED time that is incorrect, i.e. the time from first job started.
![travis_time](https://cloud.githubusercontent.com/assets/15121917/20077115/55199786-a53b-11e6-8ac4-cc24046854a7.jpg)

**Comment by BanzaiMan** [comment]
https://api.travis-ci.org/repos/betaflight/betaflight/builds/173989644 shows that the time difference between  `started_at` (when the first job started running) and `finished_at` (when the last job stopped running) is 5 minutes 17 seconds.

**Comment by AndersHoglund** [comment]
No, in reality it is 23 minutes. Sometime around 10minutes into the build, the elapsed time started over from 0. It was counting correct up to 10m. It probably got reset one more time as well, did not see that. Random resets (?).

**Comment by BanzaiMan** [comment]
Oh, I see now. https://api.travis-ci.org/jobs/173989646 started at "2016-11-07T20:31:32Z", but it's not taken into account.

**Comment by AndersHoglund** [comment]
Good, you found a clue. Thanks.

**Comment by AndersHoglund** [comment]
One symptom. When I click on a job to take a look at the log and then return back to the build overview, that is when timer is restarted from zero. I could repeat that a couple of times, but now when I try again it does not happen. Probably have to hit a time slot exactly.

**Comment by WaleedAshraf** [comment]
@BanzaiMan Any update on this issue? Im facing same problems with build times.
